lib/components/switch_view.js

```diff
@@ -19,9 +19,3 @@ SwitchView.propTypes = {
 }
 
 const NativeSwitchView = requireNativeComponent('ARSwitchView', SwitchView)
-
-export type SwitchEvent = {
-  nativeEvent: {
-    selectedIndex: number;
-  };
-};
```

Comment:
moved this into it's own class, this _should_ work, but it doesn't, see [Slack](https://artsy.slack.com/archives/mobile-banter/p1475054659000795)